docs/variants/pc_engines/hardware-matrix.md

```diff
@@ -5,6 +5,36 @@
 This document describes the hardware configurations used for validation of the
 coreboot port on the PC Engines apu2 family of mainboards.
 
+=== "apu2"
+
+    | Component                      | Description                                      |
+    |--------------------------------|--------------------------------------------------|
+    | **CPU**                        | AMD GX-412TC SOC                                 |
+    |                                | Passive cooling to enclosure                     |
+    | **RAM**                        | 4 GB DDR3 non-ECC (soldered)                     |
+    | **SSD**                        | Apacer ST170 60GB                                |
+    | **Flash memory**               | Winbond W25Q64JV                                 |
+    | **USB pendrives**              | SanDisk Ultra USB 3.0 32 GB                      |
+    | **Wireless card**              | Compex WLE900VX                                  |
+    | **Network**                    | 3x i211AT Gigabit Ethernet                       |
+    | **TPM**                        | PC Engines TPM.4A module                         |
+    | **Power Supply**               | 12V power supply                                 |
+    | **Power Control**              | RTE v1.0.0                                       |
+=== "apu3"
+
+    | Component                      | Description                                      |
+    |--------------------------------|--------------------------------------------------|
+    | **CPU**                        | AMD GX-412TC SOC                                 |
+    |                                | Passive cooling to enclosure                     |
+    | **RAM**                        | 4 GB DDR3 non-ECC (soldered)                     |
+    | **SSD**                        | Apacer ST170 60GB                                |
+    | **Flash memory**               | Winbond W25Q64JV                                 |
+    | **USB pendrives**              | SanDisk Ultra USB 3.0 32 GB                      |
```

Comment:
Which entries have been verified? Do we know what is missing in the hardware?

I see the sticks are different, looks like copy-paste from APU4:

```
bash-5.1# lsusb
Bus 002 Device 003: ID 12d1:1573 Huawei Technologies Co., Ltd. ME909u-521 mPCIe LTE/GPS card
Bus 002 Device 002: ID 0438:7900 Advanced Micro Devices, Inc. Root Hub
Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 001 Device 002: ID 0438:7900 Advanced Micro Devices, Inc. Root Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 004 Device 002: ID 0951:1666 Kingston Technology DataTraveler 100 G3/G4/SE9 G2/50
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 002: ID 0930:6545 Toshiba Corp. Kingston DataTraveler 102/2.0 / HEMA Flash Drive 2 GB / PNY Attache 4GB Stick
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```

Reply:
CPU, RAM, Flash memory, USB pendrives, Network, TPM, Power Supply.
Flash memory and Wireless cards were not verified and were at present copied.
That was regarding APU2 and APU6. Since I couldn't get APU3 to work on Thursday, no entries were verified for it.

Reply:
@pkubaj were you able to verify apu3 since then?

Reply:
Done.